Q2.txt

```diff
@@ -0,0 +1,32 @@
+//Question 2
+
+let age = 25;
+switch(age) {
```

Comment:
This switch case is fine but it can be implemented in more efficient way and less lines
eg 
```
switch(true){
 case age<=15:
   console.log("child");
}
```